src/labels/CustomLabels.labels

```diff
@@ -4932,6 +4932,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>sendAcknowledgmentNumberSent</shortDescription>
         <value>{0} acknowledgment(s) sent.</value>
     </labels>
+    <labels>
+        <fullName>sendAcknowledgmentNumberAlreadySent</fullName>
+        <categories>Send Acknowledgment Button</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>sendAcknowledgmentNumberAlreadySent</shortDescription>
+        <value>{0} acknowledgment(s) already sent.</value>
```

Comment:
@sam-knox Can you please verify the message displayed when the user attempts to send multiple acknowledgements for the same Opportunity?

Can you also review the changes in the PR?

Reply:
@patrick-yan-sf The new label needs to be included in the `package.xml`.